phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -3055,6 +3059,12 @@ public boolean isViewReferenced() {
                 tableUpsert.setString(33, physicalTableName);
             }
 
+            if (schemaVersion == null) {
+                tableUpsert.setNull(34, Types.VARCHAR);
```

Comment:
Or should have a default starting schema version?

Reply:
I'd considered it but decided against the idea because I didn't want to proscribe a particular versioning scheme...the idea is that a particular organization can choose their own version stamps (and that they don't have to if they don't want to, hence default null)